File: 1S/Primos.cpp
#include <iostream>
#include <cmath>

bool print_primes(int num, int lim);

int main(void) {
  int cter = 1;
  for(int x = 2; x < 101; x++){
    if(print_primes(x,cter) == false){
      std::cout << x << "\n";
    }
    cter++;
  }
  return 0;
}

bool print_primes(int num, int lim){
  bool chk = false;
  bool lpck = true;
  while(lpck == true){
    for(int i = 2; i < lim; i++){
      if(num % i == 0){
        chk = true;
      }
      else {
        continue;
      }
    }
    lpck = false;
  }
  if(chk == true){
    return true;
  }
  else {
    return false;
  }
}

File: 1S/Primos.py
n_sf = []
sta = 2
sto = 100

def div_prev(num,lim):
    chk = False
    aaa = True
    while aaa == True:
        for i in range(2,lim+1):
            #print("\n",num,i)
            #print("mod", num % i)
            if num % i == 0:
                #print("true")
                chk = True
            else:
                continue
                #print(false)
        aaa = False

    #print("chk", chk)
    if chk == True:
        return True
    else:
        return False

for x in range(sta,sto+1):
    if div_prev(x,len(n_sf)) == False:
        n_sf.append(x)

print(n_sf)
